db/app.controller.js

```diff
@@ -49,7 +49,12 @@ selectAllArticles(query).then((articles)=>{
 
 exports.getAllCommentsByArticleId =(req,res,next) => {
 const {article_id} = req.params
-selectCommentsByArticleID(article_id).then(comments =>{
+selectArticlesById(article_id).then((article)=>{
+if (article){
+    return selectCommentsByArticleID(article_id)
+}
```

Comment:
Your selectArticleById model will either return an article or throw an error. That means this `if` condition isn't necessary